src/metadata.rs

```diff
@@ -413,6 +414,15 @@ impl FromStr for EventArg {
                     "Expected closing `>` for `Vec`",
                 ))
             }
+        } else if s.starts_with("Option<") {
+            if s.ends_with('>') {
+                Ok(EventArg::Option(Some(Box::new(s[7..s.len() - 1].parse()?))))
```

Comment:
The `EventArg::Option(None)` is never constructed so we can remove that extra option as I mention above